test/estimate_parameters_test.cpp

```diff
@@ -7,7 +7,7 @@
 
 using namespace std;
 
-const std::string TEST_CASE_DIR = "test_cases/";
+const std::string TEST_CASE_DIR = "../../test/test_cases/";
```

Comment:
I am changing this back because if we leave `TEST_CASE_DIR = "test_cases/"`, then we have to run `pandora_test` with workdir = `SOURCE_DIR/test`. While this is fine (it is actually better IMO), it creates a bunch of temporary files in the source directory, and I think the `cmake` convention is that you always run stuff inside the build directory (i.e. `pandora_test` is inside `BUILD_DIR/test`, and we should run it inside `BUILD_DIR/test` instead of `SOURCE_DIR/test`). I am not 100% sure about this, but I think it is this.

However, I really like this change, that we refer to test cases WRT where the test source files are (i.e. `SOURCE_DIR/test`). I'd like to keep this, but I think we should symlink `SOURCE_DIR/test/test_cases/` -> `BUILD_DIR/test/test_cases/` so that we can run the test in the build dir.

I am just going for the fast and easy solution of reverting the paths back. Feel free to symlink the folder.

I might be totally wrong about this BTW...